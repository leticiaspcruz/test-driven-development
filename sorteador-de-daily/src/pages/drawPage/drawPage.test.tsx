import React from 'react';
import { render } from '@testing-library/react';
import DrawPage from './drawPage';

const mockNavigate = jest.fn();

jest.mock('react-router-dom', () => {
  return {
    ...jest.requireActual('react-router-dom'),
    useNavigate: () => mockNavigate,
  };
});

describe('test DrawPage', () => {
    test('render page correctly', () => {
        const { container } = render(<DrawPage />);
        expect(container).toMatchSnapshot();
    });
});