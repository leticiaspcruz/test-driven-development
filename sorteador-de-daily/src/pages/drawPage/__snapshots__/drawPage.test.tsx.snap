// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test DrawPage render page correctly 1`] = `
<div>
  <form>
    <input
      placeholder="Insira os nomes dos colaboradores"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
  </form>
  <ul />
  <div>
    <button
      disabled=""
    >
      Iniciar sorteio
    </button>
  </div>
</div>
`;
